test/Kestrel.InMemory.FunctionalTests/HttpsTests.cs

```diff
@@ -385,7 +385,7 @@ private class HandshakeErrorLoggerProvider : ILoggerProvider
 
             public ILogger CreateLogger(string categoryName)
             {
-                if (categoryName == nameof(HttpsConnectionAdapter))
+                if (categoryName == TypeNameHelper.GetTypeDisplayName(typeof(HttpsConnectionAdapter)))
```

Comment:
I preferred to use the `TypeNameHelper.GetTypeDisplayName` because it is used by the `LoggerFactory`. But `typeof(HttpsConnectionAdapter).FullName` would suffice for this case.